File: rmt/src/test/scala/de/flower/rmt/ui/manager/ManagerHomePageTest.scala
package de.flower.rmt.ui.manager

import de.flower.rmt.test.WicketTests
import org.testng.annotations.Test
/**
 * 
 * @author flowerrrr
 */

class ManagerHomePageTest extends WicketTests {

    @Test
    def renderPage() {
        wicketTester.startPage(new ManagerHomePage())
        wicketTester.dumpPage()
    }

}

File: rmt/src/test/java/de/flower/rmt/ui/manager/ManagerHomePageTest.java
package de.flower.rmt.ui.manager;

import de.flower.rmt.test.AbstractWicketIntegrationTests;
import org.testng.annotations.Test;

/**
 * @author flowerrrr
 */

public class ManagerHomePageTest extends AbstractWicketIntegrationTests {

    @Test
    public void testRender() {
        wicketTester.startPage(new ManagerHomePage());
        wicketTester.dumpPage();
    }
}